misc_docs/syntax/operators_triangle_pipe.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: "triangle-pipe"
+keywords: [pipe", "triangle", "operator", "function", "argument"]
+name: "|>"
+summary: "This is the `triangle pipe` operator."
+category: "operators"
```

Comment:
at some point we should probably introduce some `deprecated: true` field, to "grey out" labels in the overview maybe?